js/navbar.js

```diff
@@ -0,0 +1,38 @@
+
+var nav = document.getElementById("nav");
+var navlist = document.querySelectorAll("ul");
+var navlistArray = Array.prototype.slice.call(navlist);
```

Comment:
this works for now, but you might consider using a more specifc selector (parentNode.children) https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/children

this way if there are ever any other `ul`s in your page it won't grab those as well.